packages/graphback-codegen-client/src/helpers/writeDocuments.ts

```diff
@@ -2,15 +2,16 @@ import { existsSync, mkdirSync, writeFileSync } from 'fs';
 import { ClientTemplates } from '../templates/ClientTemplates';
 
 export function writeDocumentsToFilesystem(clientPath: string, generatedClient: ClientTemplates, extension: 'ts' | 'graphql') {
-    if (!existsSync(clientPath)) {
-        mkdirSync(clientPath, { recursive: true })
-    }
-
     const implementations: string[] = [];
     Object.keys(generatedClient).forEach((folder: string) => {
         //tslint:disable-next-line: no-any
         generatedClient[folder].forEach((c: any) => implementations.push(c.implementation));
     })
 
-    writeFileSync(`${clientPath}/graphback.${extension}`, implementations.join('\n\n'));
+    const isExtensionSpecifiedRegExp = new RegExp(/\.([a-zA-Z])+$/);
```

Comment:
This is redundant. It should be GraphQL but some people might call this .query etc.

Reply:
> This might br redundant. Filem send with GraphQL but some people might call this .query etc.
>

I don't understand what you mean here

Reply:
Sorry was typing on the phone. I do not think this is needed. We should allow people to specify whatever extension they want. Also implementation using regexp is quite inefficient

Reply:
I have removed the regexp. I'm using path now.

People can specify whatever extension they want. I only add an extension when none is specified. So, you want me to allow people specify no extension, right?

Reply:
I will leave it. Generally I tend to do less of stuff like this as it is more code to maintain etc.

Reply:
@wtrocki